lightstep/resource_metric_condition_test.go

```diff
@@ -331,7 +333,8 @@ resource "lightstep_metric_condition" "test" {
 					testAccCheckMetricConditionExists(resourceName, &condition),
 					resource.TestCheckResourceAttr(resourceName, "name", "Too many requests"),
 					resource.TestCheckResourceAttr(resourceName, "description", "A link to a playbook"),
-					resource.TestCheckNoResourceAttr(resourceName, "labels"),
```

Comment:
Any idea why we checked this before?

Reply:
I think the idea was to make sure no labels had been set, but it wasn't working properly because the name of the field in the schema is `label` rather than `labels` so this check was misleading (it wasn't doing what it was supposed to be doing). I just changed it to check the thing it was supposed to check.

Reply:
I noticed it because I was trying to model my new check after the labels one and couldn't figure out why mine was failing while this was succeeding, then realized why.